src/Auth/ApiAuthenticate.php

```diff
@@ -25,6 +25,7 @@
  * An authentication adapter for authenticating using BEdita 4 API /auth endpoint.
  *
  * @see https://docs.bedita.net/en/latest/authorization.html#authentication
+ * @deprecated 2.2.0 Use \BEdita\WebTools\Identifier\ApiIdentifier for Authentication plugin instead.
  */
 class ApiAuthenticate extends BaseAuthenticate
```

Comment:
Could we just get rid of this class? 
In 2.x branch this feature has not yet been used...

Reply:
Ok I'm going to remove it.